### PR TITLE
Header and navigation menu

### PR DESCRIPTION
**Pull request by rebekkaschaub**

### CODE REVIEW

File: frontend/src/nav/Header.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Burger from "./Burger";
+import Menu from "./Menu";
+import styled from "styled-components/macro";
```
Comment (not-enough-ram) [inline frontend/src/nav/Header.js:3]: Burger Menu... yummie

**Review by not-enough-ram — APPROVED** [review]
lgtm

File: frontend/package.json [hunk 2]
```diff
@@ -38,5 +39,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "react-router-dom": "^5.2.0"
+  }
```
Comment (fabianschmauder) [inline frontend/package.json:45]:
```suggestion
```
you already have this dependency. This should be part of the normal dependencies.
But you can add prettier here

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+import Header from "./nav/Header";
 
 export default function App() {
-  return <ResultsListPage />;
+  return (
+    <div>
+      <Header />
+
+      <Switch>
+        <Route path="/counseling" exact>
```
Comment (fabianschmauder) [inline frontend/src/App.js:12]: no landing page?

File: frontend/src/nav/Header.js (hunk 1 above)
Comment (fabianschmauder) [reply frontend/src/nav/Header.js:3]: 🍔🍟 🥤

File: frontend/src/nav/Burger.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components/macro";
+
+export default function Burger({ open, setOpen }) {
```
Comment (fabianschmauder) [inline frontend/src/nav/Burger.js:3]: 🍔 Maybe BurgerMenuIcon

**Review by fabianschmauder — COMMENTED** [review]
🥤LGTM 🍔🍟

File: frontend/src/App.js (hunk 3 above)
Comment (rebekkaschaub) [reply frontend/src/App.js:16]: Still in progress 🐌

**Pull request merged**